cupyx/_ufunc_config.py

```diff
@@ -48,23 +49,36 @@ def get_config_linalg():
     return value
 
 
+def get_config_fallback_mode():
+    try:
+        value = _config.fallback_mode
+    except AttributeError:
+        value = _config.fallback_mode = 'warn'
```

Comment:
How about setting `ignore` default for the compatibility to current behavior?
```suggestion
        value = _config.fallback_mode = 'ignore'
```

Reply:
Changed to `ignore`. But I still believe that `warn` would be a better option to notify the user where the speedup due to CuPy is not happening.

Reply:
The purpose of `fallback_mode` is "to fallback to NumPy", so "ignore" feels more natural for me. The users should set the notification as `warn` explicitly.